### PR TITLE
add fields on page payload

### PR DESCRIPTION
**Pull request by paddeo**

### CODE REVIEW

File: engine/src/main/java/org/entando/entando/aps/system/services/page/model/PageDto.java [hunk 1]
```diff
@@ -88,6 +91,11 @@ public PageDto(IPage page) {
                 .filter(widget -> widget != null)
                 .collect(Collectors.toList())
                 .size()));
+        if (null != page.getMetadata().getUpdatedAt()) {
+            SimpleDateFormat sdf = new SimpleDateFormat("yyyy-MM-dd HH:mm:ss");
```
Comment (eugeniosant) [inline engine/src/main/java/org/entando/entando/aps/system/services/page/model/PageDto.java]: Here you can use the constants SystemConstants.API_DATE_FORMAT

Comment (paddeo) [reply engine/src/main/java/org/entando/entando/aps/system/services/page/model/PageDto.java]: ok!

**Comment by entando-service** [comment]
All is well

**Comment by entando-service** [comment]
All is well

**Pull request merged**